### ISSUE TITLE
"VS Code Speech" Extension Crashes on Activation

### ISSUE THREAD

**Issue by fussbanana** [microsoft/vscode#218202]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 
- OS Version: 

### Description
The "VS Code Speech" extension for VS Code crashes as soon as it is activated. The crash occurs during the speech-to-text session after recognizing the keyword "Hey Code".

### Steps to Reproduce
1. Activate the "VS Code Speech" extension.
2. Start a keyword-recognition session.
3. Say the keyword "Hey Code".
4. The speech-to-text session starts and then crashes with an error code 0x8 (SPXERR_FILE_OPEN_FAILED).

### Expected Behavior
The extension should recognize the keyword and execute the speech-to-text functionality without errors.

### Actual Behavior
The extension recognizes the keyword, attempts to start the speech-to-text session, and then crashes with the error code 0x8 (SPXERR_FILE_OPEN_FAILED).

### Environment
- **Hardware Model:** ASUSTeK Computer INC. SABERTOOTH P67
- **Memory:** 8.0 GiB
- **Processor:** Intel® Core™ i7-2600K × 8
- **Graphics:** NVIDIA GeForce GTX 1060 6GB
- **Operating System:** Ubuntu 24.04 LTS
- **GNOME Version:** 46
- **Window Manager:** X11
- **Kernel Version:** Linux 6.8.0-35-generic

### Extension Version
- **VS Code Speech Version:** 0.10.0
- **Speech Language Pack Version:** de-de-0.4.0

### Error Log
```
2024-06-25 16:51:19.561 [trace] [vscode-speech-0] starting keyword-recognition session
2024-06-25 16:51:34.672 [trace] [vscode-speech-0] keyword recognized: Hey Code
2024-06-25 16:51:34.675 [trace] [vscode-speech-0] aborting keyword-recognition session
2024-06-25 16:51:34.753 [trace] [vscode-speech-1] starting keyword-recognition session
2024-06-25 16:51:34.754 [trace] [vscode-speech-2] starting speech-to-text session (language: de-DE, model: Microsoft Speech Recognizer de-DE FP Model V2, path: /home/sascha/.vscode/extensions/ms-vscode.vscode-speech-language-pack-de-de-0.4.0/assets/stt)
2024-06-25 16:51:34.808 [error] [vscode-speech-2] speech-to-text error: Exception with an error code: 0x8 (SPXERR_FILE_OPEN_FAILED)
[CALL STACK BEGIN]
...
[CALL STACK END]
2024-06-25 16:51:34.809 [trace] [vscode-speech-2] disposed speech-to-text session
2024-06-25 16:51:41.858 [trace] [vscode-speech-2] aborting speech-to-text session
```

### Directory Structure and Permissions
```
/home/sascha/.vscode/extensions/ms-vscode.vscode-speech-language-pack-de-de-0.4.0/assets/stt
.
├── ciphone.list
├── config.yaml
├── decoder.quantuint8.onnx
├── de-de_explicitPuncRules.txt
├── de-de_ProfanityList.enc
├── encoder.quantint8.onnx
├── joint.quantuint8.onnx
├── lid.quants8u8.onnx
├── lp.config
├── model_onnx_quant.config
├── model_sentence
├── model_word
├── prongen
│   ├── grph1031.lxa
│   ├── l1031.phn
│   ├── lsr1031.lxa
│   └── tn1031.bin
├── sr.ini
├── svad.quantized.onnx
├── tokens.list
└── version.txt
```

### Permissions
```
drwxrwxr-x 3 sascha sascha 4096 Jun 25 16:46 speechmodel.de-DE.cpu.2.0.40100365
```

### Additional Information
I have verified that all files in the directory `/home/sascha/.vscode/extensions/ms-vscode.vscode-speech-language-pack-de-de-0.4.0/assets/stt` are present and readable.

### Steps Taken
- Reinstalled the extension to ensure all files are correctly installed.
- Verified file permissions to ensure no access issues.
- The problem persists after these steps.

### Request for Assistance
Could you please help resolve this issue? Any guidance or fixes would be greatly appreciated.

Thank you!

Best regards,
[Sascha]

**Comment by fussbanana** [comment]
Hello dear development team,

I hope this message finds you well. I wanted to follow up on the status of my previously submitted bug report. Since I haven't received any feedback yet, I wanted to ensure that the report has not been overlooked.

**Comment by bpasero** [comment]
Does it work when the language is english and not german in settings?

**Comment by fussbanana** [comment]
No, that makes no difference. I have tried all sorts of variations. It's
probably not because of the German language package.


Benjamin Pasero ***@***.***> schrieb am Sa. 29. Juni 2024 um
08:12:

**Comment by fussbanana** [comment]
Yes, I have also tested without the German language pack. It stops working
with the same message.


Benjamin Pasero ***@***.***> schrieb am Sa. 29. Juni 2024 um
08:12:

**Comment by bpasero** [comment]
How about the latest version of the speech extension?
